ajax/libs/jquery-watch/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "jquery-watch",
+  "description": "A jQuery plug-in to watch CSS style and attribute changes and get notified when a change occurs",
```

Comment:
Let's modify the description as the same as upstream 
`"This small jQuery plug-in allows you to monitor changes to any DOM element's CSS styles, attributes or properties and fires a callback in response to any change in the monitored styles or attributes."`
We'd like this field as the same with upstream. :)
Thanks.

ref: https://github.com/RickStrahl/jquery-watch/blob/master/package.json#L4